index.test.js

```diff
@@ -0,0 +1,83 @@
+const skeletor = require('./index');
+
+const validConfig = {
+	tasks: [
+		{
+			name: 'task1',
+			subTasks: [
+				{
+					name: 'subTask1',
+					plugins: []
+				},
+				{
+					name: 'subTask2',
+					plugins: []
+				},
+				{
+					name: 'subTask3',
+					plugins: []
+				}
+			]
+		}
+	]
+};
+
+const validResponse = {
+	taskName: 'task1',
```

Comment:
Spacing seems a little wonky here

Reply:
Which spacing are you referring to?  I'm not seeing anything out of the ordinary.

Reply:
I see eight spaces on taskName and 4 on the rest. May have mix of tab and spaces.

Reply:
Honestly, if you don't see it and the linter doesn't catch it, it's probably fine. Github has been doing weird formatting on me lately

Reply:
Yeah I'm not seeing it.  Just looks like a single tab indent to me in Sublime.